interceptors/logging/interceptors.go

```diff
@@ -45,42 +48,94 @@ func (c *reporter) PostCall(err error, duration time.Duration) {
 	code := c.opts.codeFunc(err)
 	logger = logger.With("grpc.code", code.String())
 	if err != nil {
-		logger = logger.With("error", fmt.Sprintf("%v", err))
+		logger = logger.With("grpc.error", fmt.Sprintf("%v", err))
+	}
+
+	logger = logger.With(c.opts.durationFieldFunc(duration)...)
+	logger = logger.With("protocol", "grpc")
+	logger.Log(c.opts.levelFunc(code), fmt.Sprintf("finished_%s_%s_call", c.kind, c.typ))
```

Comment:
I think this is not what users might expect.

When you see some decision like this, especially such "controversial" (human readable streaming being with `_`) (: instead of going for it and spending time to adjust every PR, let's discuss first.

It's also clear like we need to common and consistent design on what such message we want to make this decision. I believe we could extend our proposal with such section. WDYT? (:

Reply:
Remove `_` from the messages

Reply:
Add the key as `kind = c.kind` and `type = c.type`, and at the end log the `finished call`

Reply:
`msg = finished call`

Reply:
Changed